docs/src/pages/en/get-started.mdx

```diff
@@ -4,63 +4,79 @@ description: Docs intro
 
 ## Key Features
 
-- **Web Components:** Compatible with any Javascript framework (React, Angular, Svelte, etc.)
-- **Unified API:** Compatible with the `<video>` and `<audio>` elements and a lot of other players
-- **HTML first:** Write plain HTML to add and remove controls
-- **Easy styling:** Simple CSS to style the controls
+- **Web Components:** Use with or without a JavaScript framework, compatible with: React, Angular, VueJS, Svelte, and more.
+- **Styling:** Use simple CSS to style components
+- **Theming:** Build custom themes that group together controls and style
+- **Accessible:** All controls express their state in an accessible way
+- **Unified API:** Compatible with standard `<video>` and `<audio>` elements and many other media elements
 
-## _Quick_ Demo
+## Example usage
 
-### Adding controls to a video element
-
-Just HTML. No javascripting required.
-
-[![Add controls to the media](https://image.mux.com/fbrV01YkyMrnp01BItrdWL029IurCj2gxlS/animated.gif?width=640&end=10&fps=15)](https://stream.new/v/fhRqeSkHntHb2IZ3AyduTK02l3b9j1EyZTjGHdyERg018)
+<SandpackContainer
+  html={`<media-controller>
+  <video
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+    playsinline
+    muted
+  ></video>
+ <media-control-bar>
+    <media-play-button></media-play-button>
+    <media-mute-button></media-mute-button>
+    <media-time-range></media-time-range>
+    <media-fullscreen-button></media-fullscreen-button>
+  </media-control-bar>
+</media-controller>`}
+/>
 
 ## Install
 
-`<media-chrome>` is only packaged as a javascript module (es6), which is supported by all evergreen browsers and Node v12+. The package includes all of the existing media controls.
+Media Chrome is packaged as an ES6 JavaScript module which is supported by all evergreen browsers and Node v12+.
 
 ### Option 1: Hosted
 
 Load the module in the `<head>` of your HTML page. Note the `type="module"`, that's important.
 
-> Modules are always loaded asynchronously by the browser, so it's ok to load them in the head 👍
-
 <CopyToClipboard>
 ```html
 <script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@0/+esm"></script>
 ```
 </CopyToClipboard>
 
+> Modules are always loaded asynchronously by the browser, so it's ok to load them in the head 👍
+
 ### Option 2: Bundled via npm
 
+Install via npm (you could also use Yarn or your favorite package manager)
+
 <CopyToClipboard>
 ```bash
 npm install media-chrome --save
 ```
 </CopyToClipboard>
 
+Import into your app.
+
 <CopyToClipboard>
 ```js
 import 'media-chrome';
 ```
 </CopyToClipboard>
 
-Include in your app javascript (e.g. src/App.js)
-This will register the custom elements with the browser so they can be used as HTML.
+Importing the package like this will automatically register all of the available components in the browser. This means you can add a control like `<media-play-button>` to your page.
```

Comment:
suggestion(non-blocking): Since this doesn't actually pull in everything (theme component & themes, extras, experimental), maybe:

```suggestion
Importing the package like this will automatically register all of the available of the core Media Chrome components in the browser. This means you can add a control like `<media-play-button>` to your page.
```

Reply:
I'll update this, your edit suggestion has a double "of the" in the sentence.